src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -46,6 +46,20 @@
 			"path": "$.holdings.metadata.updatedDate",
 			"recordType": "HOLDINGS"
 		},
+    {
+      "fieldId":"instance.alternativetitles.cover.title",
+      "displayNameKey":"instance.alternativeTitles",
+      "referenceDataValue":"Cover title",
+      "path":"$.instance[*].alternativeTitles[?(@.alternativeTitle==Cover title)]",
```

Comment:
yea I think this will not work because , I can give any title I like
example:
` "alternativeTitles" : [ {
      "alternativeTitleTypeId" : "5c364ce4-c8fd-4891-a28d-bb91c9bcdbfb",
      "alternativeTitle" : "kv"
    }, {
      "alternativeTitleTypeId" : "09964ad1-7aed-49b8-8223-a4c105e3ef87",
      "alternativeTitle" : "testing alt"
    } ],`

Reply:
agree, thank you, updated the field